api/src/args.rs

```diff
@@ -39,6 +39,12 @@ pub struct Args {
         default_value = "5001"
     )]
     pub(crate) provisioner_port: Port,
+    #[structopt(
+        long,
+        about = "MSSV - Minimum supported Shuttle Version",
+        default_value = shuttle_service::VERSION
+    )]
+    pub(crate) shuttle_version: String,
```

Comment:
It seems to me the first time this is validated as a valid `semver::Version` is when it's parsed in the `get_shuttle_service_from_user_crate` function, which would only happen by issuing a user error whereas the error is actually on our end.

Would it not be better to have this `: String` be replaced by `: semver::Version` so that it gets validated there? I believe this should work since `semver::Version` is `FromStr`.

Reply:
@brokad it's working! Thanks for that